qubes-trust-daemon.cpp

```diff
@@ -141,17 +140,14 @@ void mark_files_as_untrusted(const std::set<std::string> file_paths) {
                 execv("/usr/bin/qvm-file-trust", (char**) qvm_argv);
 
                 // Unreachable if no error
-                perror("execl qvm-file-trust");
-                exit(1);
+                perror("execl qvm-file-trust failed");
             case -1:
                 // Fork failed
                 perror("fork failed");
-                exit(1);
```

Comment:
break, or better return (to not clear `untrusted_buffer`)

Reply:
Changed in 5aa523b